src/fragments/reference.js

```diff
@@ -0,0 +1,18 @@
+import { graphql } from 'gatsby';
+
+export const imagesFragment = graphql`
+  fragment Images on PagesJsonReference {
+    images {
+      mediaPath
+      altText
+      figText
+    }
+  }
+`;
+
+export const referenceFragment = graphql`
```

Comment:
will need to add other fragments for other blocks as they are needed, can't be added right now or it will error from being not found in any query.